### DISCUSSION TITLE
Elsa3 add webhooks is  not work

### DISCUSSION THREAD

**Discussion by Godaday** [elsa-workflows/elsa-core#6009] [Q&A]
I tried to add the webhooks module in Elsa3.2, but it didn't work. I need help. 
Here are my Service and BlzorWasm.
![image](https://github.com/user-attachments/assets/fbf786c7-ae0c-4be5-b640-a95ef3c91e05)
![image](https://github.com/user-attachments/assets/76a80e46-dc66-424d-954d-21415291a874)
![image](https://github.com/user-attachments/assets/fb3c2e0b-ca25-4b9d-8bf2-0beeecfca41d)
Click WebHooks in the menu and it will be a blank page
![image](https://github.com/user-attachments/assets/5d581df4-3214-4e3d-a602-e2467c1e886a)

**Comment by tse-wei-chen** [comment]
I have the same issue. Is there any progress?

**Comment by sfmskywalker** [comment]
Thanks for the report. I can confirm that the Webhooks Studio page was incomplete in this generation of the feature: the menu route rendered only the page heading, with no sink-management UI behind it. So the blank page was not simply a configuration mistake in your application.

The missing Webhook Sinks management UI and API client were implemented later here:

https://github.com/elsa-workflows/elsa-extensions/commit/579dbdcf3e3629503a38ebc8d44cc8cba06a99c4

Please try the latest available matching Elsa server and Studio packages. If the page is still blank there, open a new report with the exact package versions plus the browser console and network errors so we can diagnose the current behavior rather than the old 3.2 setup.

**Comment by sfmskywalker** [comment]
The original blank Webhooks page in Elsa 3.2 is resolved by the later Webhook Sinks management implementation. Closing this historical report as resolved; if the problem persists with matching current server and Studio packages, please open a new report with the package versions and browser console/network errors.
